prov/rxd/src/rxd.h

```diff
@@ -284,7 +286,14 @@ struct rxd_op_pkt {
 	struct rxd_pkt_hdr	pkt_hdr;
 
 	uint64_t		num_segs;
-	uint64_t		tag;
+	union {
+		uint64_t		tag;
+		struct {
+			uint64_t		iov_count;
+			struct ofi_rma_iov	rma[RXD_IOV_LIMIT];
```

Comment:
I'm working on a follow up patch that separates all op types in different op_pkt structs to save space because of this. (I'm working on atomics right now which adds to this problem). Want me to keep it in the follow up patch or finish the patch and put it with this PR?

Reply:
A follow up patch is fine.